src/handlers/comment/handlers/ask.ts

```diff
@@ -0,0 +1,218 @@
+// import { CreateChatCompletionRequestMessage, Configuration, OpenAIApi, ResponseTypes } from 'openai-edge'
+import { getBotContext, getLogger } from "../../../bindings";
+import { Payload, UserType } from "../../../types";
+import { getAllIssueComments } from "../../../helpers";
+import { BasePromptTemplate, PromptTemplate } from "langchain";
+
+import OpenAI from "openai";
+import { CreateChatCompletionRequestMessage } from "openai/resources/chat";
+
+const openAI = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY,
```

Comment:
should read apiKey from getConfig() you already did it in src/bindings/config.ts